src/test/scala/cognite/spark/v1/TimeSeriesRelationTest.scala

```diff
@@ -936,6 +936,83 @@ class TimeSeriesRelationTest extends FlatSpec with Matchers with SparkTest with
     assert(descriptionsAfterUpdate.length == 5)
   }
 
+  it should "support ignoring unknown ids in deletes" in {
+    val unit = "ignore-unknown-id-test"
+    cleanUpTimeSeriesTestDataByUnit(unit)
+    val dfWithDeletesAsUnit = retryWhile[Array[Row]](
+      spark.sql(s"select * from destinationTimeSeries where unit = '$unit'").collect,
+      df => df.length > 0)
+    assert(dfWithDeletesAsUnit.length == 0)
+
+    // Insert some test data
+    spark
+      .sql(s"""
+              |select "foo" as description,
+              |name,
+              |false as isString,
+              |map("bar", "test") as metadata,
+              |'$unit' as unit,
+              |null as assetId,
+              |false as isStep,
+              |null as securityCategories,
+              |bigint(0) as id,
+              |'delete example' as externalId,
+              |now() as createdTime,
+              |now() lastUpdatedTime,
+              |dataSetId
+              |from sourceTimeSeries
+              |limit 1
+     """.stripMargin)
+      .select(destinationDf.columns.map(col): _*)
+      .write
+      .insertInto("destinationTimeSeries")
+
+    // Check if insert worked
+    val idsAfterInsert =
+      retryWhile[Array[Row]](
+        spark
+          .sql(s"select id from destinationTimeSeries where unit = '$unit'")
+          .collect,
+        df => df.length < 1)
+    assert(idsAfterInsert.length == 1)
```

Comment:
It seems to me that you don't need to insert the test data at all.

Reply:
I tried following the format used for Assets and events deletion (It had insertion of test data following the deletes).
Also, I don't know why "timeseries not found " error ie I guess error 400 is not thrown if there's no insertion.

Reply:
The tests in assets and events for ignoreUnknownIds are not very good, it would be useful if they tested a delete on the created id+unknown id (which should fail without the option, and leave created id present), then did a delete on with ignoreUnknownIds set (which should succeed, and the created id should be gone), but they're not doing that.

You can just keep the other tests (attempt to delete a non-existing id with ignoreUnknownIds should be ok, should throw an exception without the setting). You can change the non-existing id to something more obviously non-existing, like 123.

Other than that, this PR looks good to me, assuming it's using the correct SDK version.

Reply:
What I understand from the conversation that the deletes are performed by different executors and hence performing deletes with (created + unknown)id with and without (ie twice) unknown id setting won't make sense because the created id would be already deleted in the first go, right?

Also, I guess I have not updated the SDK version to the one which would be released now ( I guess we should release the updated scala SDK)